package/yast2-network.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-network
-Version:        3.1.174
+Version:        3.1.173.1
```

Comment:
Just for the records, the changes file looks fine. This line fixes a problem in #491.